### ISSUE TITLE
CDA.PL

### ISSUE THREAD

**Issue by fuqet90** [yt-dlp/yt-dlp#7773]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Poland

### Provide a description that is worded well enough to be understood

Can't download any videos.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (zip)
[debug] Python 3.9.2 (CPython x86_64 64bit) - Linux-5.10.0-23-amd64-x86_64-with-glibc2.31 (OpenSSL 1.1.1n  15 Mar 2022, glibc 2.31)
[debug] exe versions: ffmpeg 4.3.6-0, ffprobe 4.3.6-0, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.9.7, brotli-1.0.9, certifi-2020.06.20, mutagen-1.45.1, pyxattr-0.7.2, sqlite3-2.6.0, websockets-8.1
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: cedf44a9cf3c241520a692c1006d276e6f434d4d945727c47cb1a70d32953f4a
yt-dlp is up to date (stable@2023.07.06)
```

**Comment by bashonly** [comment]
post actual verbose log of download attempt

**Comment by fuqet90** [comment]
WARNING: [CDA] unable to extract default player_json; please report this issue o  n  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issu  e template. Confirm you are on the latest version using  yt-dlp -U
ERROR: [CDA] 10997593ce: No video formats found!; please report this issue on  h  ttps://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue te  mplate. Confirm you are on the latest version using  yt-dlp -U

**Comment by dirkf** [comment]
See also https://github.com/ytdl-org/youtube-dl/issues/30873 (not apparently the same issue though).

**Comment by dirkf** [comment]
This is an age-confirm page but with a different structure from that expected:
```console
$ yt-dlp -v -F 'https://www.cda.pl/video/1553664551'
[debug] Command-line config: ['-v', '-F', 'https://www.cda.pl/video/1553664551']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.06.22 [812cdfa06] (source)
[debug] Lazy loading extractors is disabled
[debug] Git HEAD: de4cf77ec
[debug] Python 3.9.16 (CPython i686 32bit) - Linux-4.4.0-210-generic-i686-with-glibc2.23 (OpenSSL 1.1.1u  30 May 2023, glibc 2.23)
[debug] exe versions: ffmpeg 4.3, ffprobe 4.3
[debug] Optional libraries: Cryptodome-3.11.0, certifi-2019.11.28, secretstorage-3.2.0, sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1851 extractors
[CDA] Extracting URL: https://www.cda.pl/video/1553664551
[CDA] 1553664551: Downloading webpage
WARNING: [CDA] unable to extract default player_json; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
ERROR: [CDA] 1553664551: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/home/df/Documents/src/yt-dlp/yt_dlp/YoutubeDL.py", line 1555, in wrapper
    return func(self, *args, **kwargs)
  File "/home/df/Documents/src/yt-dlp/yt_dlp/YoutubeDL.py", line 1652, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/home/df/Documents/src/yt-dlp/yt_dlp/YoutubeDL.py", line 1711, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "/home/df/Documents/src/yt-dlp/yt_dlp/YoutubeDL.py", line 2653, in process_video_result
    self.raise_no_formats(info_dict)
  File "/home/df/Documents/src/yt-dlp/yt_dlp/YoutubeDL.py", line 1067, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [CDA] 1553664551: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

$
```
Page JS sets cookie `adult=1` but doing that and refetching doesn't help.

A single resolution is available by fetching the `embedUrl` from the ld+json and decoding an attribute of the player element there.

Of the 3 extractor tests 1 and 2 don't show this problem. There's no `view_count` in test 2. Test 3 is 410, but a page like the above could replace it once the extraction works.

**Comment by fuqet90** [comment]
Hi.

That you for the reply.

The problem is with age confirmation in modal window.

I had to add credentials to my free account to be able to bypass this window.

**Comment by bashonly** [comment]
Duplicate of #5980

Will be fixed by #6638
